dev/sparktestsupport/modules.py

```diff
@@ -743,6 +743,20 @@ def __hash__(self):
     ]
 )
 
+docker_integration_tests = Module(
+    name="docker-integration-tests",
+    dependencies=[],
+    build_profile_flags=["-Pdocker-integration-tests"],
+    source_file_regexes=["external/docker-integration-tests"],
+    sbt_test_goals=["docker-integration-tests/test"],
+    environ={
+        "ENABLE_DOCKER_INTEGRATION_TESTS": "1"
+    },
```

Comment:
@sarutak can we add this environment variable if `GITHUB_ACTION` environment variable is set? e.g.)

```suggestion
    environ=None if "GITHUB_ACTIONS" in os.environ else {
        "ENABLE_DOCKER_INTEGRATION_TESTS": "1"
    },
```

Reply:
@sarutak BTW, I added this suggestion a bit after a quick testing .. now this should work correctly ... hopefully ..

Reply:
Thanks. I tried locally and seems to work.